support-frontend/assets/pages/paper-subscription-checkout/components/paperCheckoutForm.tsx

```diff
@@ -373,7 +351,7 @@ function PaperCheckoutForm(props: PropTypes) {
 							id="title"
 							label="Title"
 							optional
-							value={props.title}
+							value={props.title ? props.title : ''}
```

Comment:
ternary used as prop type can be of type null, which cannot be assigned to `value`

Reply:
could use `props.title ?? ''`

Reply:
could do `props.title ?? ''`